apps/dcos_l4lb/src/dcos_l4lb_network_sup.erl

```diff
@@ -26,25 +26,25 @@
 %% ===================================================================
 
 start_link() ->
-  supervisor:start_link({local, ?MODULE}, ?MODULE, []).
-
+    supervisor:start_link({local, ?MODULE}, ?MODULE, []).
 
 maybe_ipvs_child() ->
-  case dcos_l4lb_config:networking() of
-    true ->
-       [?CHILD(dcos_l4lb_mgr, worker)];
-    false ->
-      []
-  end.
+    case dcos_l4lb_config:networking() of
+        true ->
+            [?CHILD(dcos_l4lb_mgr, worker)];
+        false ->
+            []
+    end.
 
 init([]) ->
-  Children = maybe_ipvs_child () ++ [
-    ?CHILD(dcos_l4lb_lashup_vip_listener, worker)
-  ],
-  {ok,
-    {
-      {rest_for_one, 5, 10},
-      Children
-    }
-  }.
+    dcos_l4lb_lashup_vip_listener:init_metrics(),
+    Children = maybe_ipvs_child () ++ [
+        ?CHILD(dcos_l4lb_lashup_vip_listener, worker)
+    ],
+    {ok, {#{strategy => rest_for_one}, Children}}.
+        {
+            {rest_for_one, 5, 10},
```

Comment:
Why have we changed `intensity` from 5 to 1 (default) and `period` from 10 to 5 (default)?

Reply:
My guess it's default values from a very old supervisor template

https://github.com/rebar/rebar/blob/master/priv/templates/simpleapp_sup.erl#L26